notifications.json

```diff
@@ -0,0 +1,861 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "version": "1.0",
+        "title": "Notification API",
+        "description": "Notification API"
+    },
+    "host": "e2effzjxvtgbmuilw-mock.stoplight-proxy.io",
+    "basePath": "/v1",
+    "schemes": [
+        "https"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "paths": {
+        "/Messages/{id}": {
```

Comment:
[Zalando REST APIs policies](http://zalando.github.io/restful-api-guidelines/json-guidelines/JsonGuidelines.html) are very good indeed but also VERY opinionated. 

ie. there are some directives like "**Must**: Do Not Use URI Versioning" which IMHO are hard to be applied blindly.

Moreover, **if** we want to be compliant and coherent with [SCIM API specs](https://tools.ietf.org/html/rfc7644) we have to start with a different style (ie. camelCased endpoints and so on).